notebooks/HASP/COSLifetimePositions/COSLifetimePositions.ipynb

```diff
@@ -0,0 +1,965 @@
+    "* `astroquery.mast.Observations` to retrieve COS spectra from MAST\n",
+    "* `glob` to work with multiple files in our directories\n",
+    "* `matplotlib.pyplot` to plot spectra\n",
+    "* `numpy` to calculate various values\n",
+    "* `os` to interact with the operating system\n",
+    "* `pathlib.Path` to create a data directory \n",
+    "* `scipy.optimize.curve_fit` to fit a function to a spectrum\n",
+    "* `shutil` to perform directory and file operations\n",
+    "* `specutils.Spectrum1d` to create Spectrum1d objects\n",
+    "\n",
+    "We recommend creating a HASP-specific `conda` environment. You can check out our [Setup.ipynb](https://github.com/spacetelescope/hst_notebooks/tree/main/notebooks/HASP/Setup) notebook to create such an environment. Alternatively, you can also download the required dependencies to run this notebook with the terminal command: \n",
+    "\n",
+    "`pip install -r requirements.txt`\n",
+    "\n",
+    "Let's import all of the packages that we will use in this notebook by running the next cell:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4334aa1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import glob\n",
+    "import matplotlib\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import shutil\n",
+    "\n",
+    "from astropy.convolution import convolve\n",
+    "from astropy.io import ascii, fits\n",
+    "from astropy.visualization import quantity_support\n",
+    "from astroquery.mast import Observations\n",
+    "from matplotlib import pyplot as plt\n",
+    "from pathlib import Path\n",
+    "from scipy.optimize import curve_fit\n",
+    "\n",
+    "# Specify plot parameters.\n",
+    "quantity_support()  # to put units on the axes below \n",
+    "matplotlib.rcParams['figure.figsize'] = [15, 5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6dd2dbfd",
+   "metadata": {},
+   "source": [
+    "<a id = lsfdownload></a>\n",
+    "## 1. Download the COS Line-Spread Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bdca358",
+   "metadata": {},
+   "source": [
+    "While it is possible to download the various COS line-spread functions via Python, it is easier to retrieve them by hand.  From the [COS homepage](https://www.stsci.edu/hst/instrumentation/cos/performance/spectral-resolution), follow the link to the [COS Line Spread Functions](https://www.stsci.edu/hst/instrumentation/cos/performance/spectral-resolution).  Scroll down to the tables containing the various LSFs.  For each lifetime position (1 to 5), retrieve the LSF for grating G130M and central wavelength 1291.  Be sure to get the \"Tabulated Theoretical Line Spread Functions\" on the left side of the page, NOT the \"Cross Dispersion\" functions on the right.  The files should have names like \"aa_LSFTable_G130M_1291_LP1_cn.dat.\"  Store them in a subdirectory called \"LSF\".\n",
```

Comment:
this step needs to be done in python, or else the CI will fail because the rest of the notebook depends on these files. Write a cell that creates the LSF subdirectory in the expected location, and downloads each of these files to that directory.